tests/util/verify.sh

```diff
@@ -131,7 +131,10 @@ __cmp_like() {
     local out="${1//$'\r'}"
     local expected=$2
     local ipregex="^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$"
-    local timeregex="^([0-9]+[smhd])+$"
+    local durationregex="^([0-9]+[smhd])+$"
+    local versionregex="^[0-9]+\.[0-9]+\.[0-9]+$"
+    local dateregex="^\d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$"
+    local timeregex="^(2[0-3]|[01]?[0-9]):([0-5]?[0-9]):([0-5]?[0-9]).[0-9]+$"
```

Comment:
Would be good to add these new checks to the `_verify_like` doc (in `verify.sh` and `tests/README.md`)

Reply:
done